plugin/parser/differ/mysql/differ.go

```diff
@@ -394,17 +397,20 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 	}
 
 	var buf bytes.Buffer
-	_, _ = buf.Write([]byte("SET FOREIGN_KEY_CHECKS=0;\n"))
-
+	_, _ = buf.Write([]byte(disableFKCheckStmt))
```

Comment:
Instead of writing it here, we could return fmt.Sprintf("%s\n%s", disableFKCheckStmt, buf.String()) at the end of the function.

Reply:
Done.